sv_pipeline/genome/load_data.py

```diff
@@ -59,7 +59,7 @@
     'samples_num_alt_2': lambda rows: get_sample_num_alt_x(rows, 2),
 }
 
-SAMPLES_QS_FIELDS = {'samples_qs_{}_to_{}'.format(i, i+QS_BIN_SIZE): i for i in range(0, 1000, QS_BIN_SIZE)}
+SAMPLES_QS_FIELDS = {'samples_gq_sv_{}_to_{}'.format(i, i+QS_BIN_SIZE): i for i in range(0, 1000, QS_BIN_SIZE)}
```

Comment:
you need to rename all the constants that have `QS` in them, otherwise its really confusing since qs is in neither the input nor the output